### PR TITLE
[YUNIKORN-2945] Add punctuation for better clarity in Scheduler Configuration

### PR DESCRIPTION
**Pull request by kousei47747**
### What is this PR for?
Add punctuations for typos and better clarity.

Changes:
1. "pre emption" >> "preemption"
2. "user based" >> "user-based"
3. "cluster-wide" >> "cluster wide"
4. "In other words when the access control list of a queue does not allow access the parent queue is checked." >>
"In other words, when the access control list of a queue does not allow access, the parent queue is checked."    

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2945

### CODE REVIEW

File: docs/design/scheduler_configuration.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ On each queue, except the root queue, the following properties can be set:
 * Queue Permissions:
     * SubmitACL (ACL)
     * AdminACL (ACL)
-* Pre emption setting:
+* Pre-emption setting:
```
Comment (craigcondit) [inline docs/design/scheduler_configuration.md:98]: Preemption is one word, don't hyphenate it.

**Review by craigcondit — APPROVED** [review]
Good except for one minor point.

**Review by craigcondit — CHANGES_REQUESTED** [review]
See above.

**Comment by kousei47747** [comment]
@craigcondit thanks for reviewing!
I've removed the hyphens in preemption, please check.

**Review by chenyulin0719 — APPROVED** [review]
+1 LGTM

**Comment by chenyulin0719** [comment]
Merged. 

@kousei47747 Thanks for fixing it. Just a gentle reminder, for the next PR, please remember to cleanup the unrelated commits before submitting the new PR.

**Pull request closed without merging**